src/messages.cs

```diff
@@ -130,6 +144,26 @@ interface MSConversation
 		[Export ("insertAttachment:withAlternateFilename:completionHandler:")]
 		[Async]
 		void InsertAttachment (NSUrl url, [NullAllowed] string filename, [NullAllowed] Action<NSError> completionHandler);
+
+		[iOS (11,0)]
+		[Export ("sendMessage:completionHandler:")]
+		[Async]
+		void SendMessage (MSMessage message, [NullAllowed] Action<NSError> completionHandler);
```

Comment:
This match the existing `Insert*` API so we better use the longer names.

That's also a case that could bite back, e.g. `sendFoo:` and `sendBar:` both accepting an `NSString` with a different meaning (but the C# signature would be identical). It's easier to add overloads when the first parameters are identical.